### PR TITLE
[SAC-206][TEST] Add scoverage

### PR DESCRIPTION
**Pull request by Bongani**
## What changes were proposed in this pull request?

In order to help with increasing unit test coverage in the future, SCoverage code coverage report plugin has been included in the project. 

```
$ mvn scoverage:report
$ open spark-atlas-connector/target/site/scoverage/overview.html
```
report

## How was this patch tested?

Manual testing using command "mvn scoverage:report" to produce html files

### CODE REVIEW

**Comment by Bongani** [comment]
@dongjoon-hyun I have removed the Cobertura code coverage plugin from the other pull request and created this pull request as you had suggested.

**Comment by dongjoon-hyun** [comment]
@Bongani . Did you try to use `Scoverage` before?

**Comment by Bongani** [comment]
> @Bongani . Did you try to use `Scoverage` before?

@dongjoon-hyun no. I actually didn't know there was a mvn plugin, but I see there is one. Is it preferred over Cobetura? I can give it a try near the end of this weekend.

File: pom.xml [hunk 1]
```diff
@@ -540,6 +542,20 @@
           </execution>
         </executions>
       </plugin>
+
+        <plugin>
```
[the remark this answers is not in the document]
Comment (Bongani) [reply pom.xml]: Fixed indentation

**Comment by Bongani** [comment]
@dongjoon-hyun I tested SCoverage and it worked. I have updated the plugin, and changed the comments on the pull request.

**Comment by dongjoon-hyun** [comment]
Thank you so much, @Bongani . I verified your PR and updated the PR title and description.

cc @HyukjinKwon , @HeartSaVioR .

**Pull request merged**